src/browser/modules/Sidebar/Settings.jsx

```diff
@@ -81,6 +81,13 @@ const visualSettings =
             displayName: 'Max Rows',
             tooltip: "Max number of rows to render in 'Rows' result view"
           }
+        },
+        {
+          autoComplete: {
+            displayName: 'Auto Complete',
```

Comment:
I think the suggestion on the example: `Connect result nodes` is a better description that makes more sense.